spec/wwtd_spec.rb

```diff
@@ -16,7 +16,7 @@ def ignore_ci_errors!(result)
     end
 
     def bundle(extra="")
-      Bundler.with_clean_env { sh "bundle #{extra}" }
+      Bundler.with_unbundled_env { sh "bundle #{extra}" }
```

Comment:
maybe just call WWTD.with_unbundled_env here to avoid duplication in case this breaks the build